src/solver/optimisation/opt_appel_solveur_quadratique.cpp

```diff
@@ -183,3 +183,153 @@ bool OPT_AppelDuSolveurQuadratique(PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudr
         return false;
     }
 }
+
+void setInteriorPointProblem(PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre,
+                             PROBLEME_POINT_INTERIEUR& Probleme)
+{
+    double ToleranceSurLAdmissibilite;
+    int ChoixToleranceParDefautSurLAdmissibilite;
+    double ToleranceSurLaStationnarite;
+    int ChoixToleranceParDefautSurLaStationnarite;
+    double ToleranceSurLaComplementarite;
+    int ChoixToleranceParDefautSurLaComplementarite;
+
+    ToleranceSurLAdmissibilite = 1.e-5;
+    ChoixToleranceParDefautSurLAdmissibilite = OUI_PI;
+
+    ToleranceSurLaStationnarite = 1.e-5;
+    ChoixToleranceParDefautSurLaStationnarite = OUI_PI;
+
+    ToleranceSurLaComplementarite = 1.e-5;
+    ChoixToleranceParDefautSurLaComplementarite = OUI_PI;
+
+    Probleme.NombreMaxDIterations = -1;
+    Probleme.CoutQuadratique = ProblemeAResoudre->CoutQuadratique;
+    Probleme.CoutLineaire = ProblemeAResoudre->CoutLineaire;
+    Probleme.X = ProblemeAResoudre->X;
+    Probleme.Xmin = ProblemeAResoudre->Xmin;
+    Probleme.Xmax = ProblemeAResoudre->Xmax;
+    Probleme.NombreDeVariables = ProblemeAResoudre->NombreDeVariables;
+    Probleme.TypeDeVariable = ProblemeAResoudre->TypeDeVariable;
+
+    Probleme.VariableBinaire = (char*)ProblemeAResoudre->CoutsReduits;
```

Comment:
double* -> char* casting, avoid this. You could create binary variables by accident.

double -> char. In most cases, the char value will be equal to PI_OUI=1. Allocate a real array and set all values to `PI_NON`.